core/seatMint.js

```diff
@@ -0,0 +1,62 @@
+import harden from '@agoric/harden';
+
+import { makeSeatConfigMaker } from './config/seatConfig';
+import { makeMint } from './issuers';
+
+/**
+ * `makeSeatMint` creates an instance of the seatMint with an
+ * associated WeakMap mapping ids (represented by unique empty
+ * objects) to use objects
+ */
+const makeSeatMint = (description = 'seats') => {
+  const idObjsToSeats = new WeakMap();
+
+  const addUseObj = (idObj, useObj) => {
+    idObjsToSeats.set(idObj, useObj);
+  };
+
+  const makeUseObj = seatQuantity => {
```

Comment:
It's surprising that access to the quantity gives access to the use Object. In our other contracts, the quantity is used as part of the description of what's being exchanged. wouldn't we want to be able to share the quantity of an invitation in order to discuss what the invitation is for?

Reply:
I think there's a misunderstanding here. Access to the quantity doesn't give access to the use object, so we can share the quantity freely.

Reply:
Ah, I see. makeUseObj() is an internal function, so no one external can use it. Thanks for the correction.